src/main/java/com/github/manolo8/darkbot/backpage/auction/AuctionItems.java

```diff
@@ -0,0 +1,75 @@
+package com.github.manolo8.darkbot.backpage.auction;
+
+public class AuctionItems {
+    protected String id, lootID, name, type, highest, current, you, instantBuy;
```

Comment:
About this class in general:

Things that are not strings, should not be represented as strings (most importantly, amounts)

getters should NOT be one-line, use proper line breaks and formatting

"highest", "current" and "you" are exremely unspecific:
highestBid, currentBid, ownBid may make more ense

What is the difference between current and own? aren't there just 2 numbers? one is the highest (and current) bid, and the other is your bid?

Reply:
agreed on the amount should not be represented as string,
have been updated to to highestBidder(person name), currentBid and ownBid